domains/kingltn.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "Kingltnn"
+    },
+    "record": {
+        "CNAME": "https://kingltnn.github.io/"
```

```suggestion
        "CNAME": "kingltnn.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).